comps/lvms/src/integrations/opea_llama_vision_lvm.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Need to connect implement with test scripts, so the code change will trigger related test.    
`opea_llama_vision_lvm.py` -> `opea_llama_vision.py`
The test name should be `test_lvms_opea_llama_vision.sh`

Reply:
OK, thanks.